torch/csrc/jit/script/sugared_value.cpp

```diff
@@ -29,6 +29,42 @@ std::shared_ptr<SugaredValue> PrintValue::call(
   return std::make_shared<NoneValue>();
 }
 
+std::shared_ptr<SugaredValue> SortedValue::call(
```

Comment:
I think most of this can be implemented in operators as @driazati suggested, you can just modularized/refactor the functions in register_prim_ops and implement the same thing for `sorted` as a `builtinFunction`.

Reply:
if it were implemented in operators i would have to re-implement all of the schema checking within `register_prim_ops` that exists for `aten::sort` [here](https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/register_prim_ops.cpp#L2553) which isn't very maintainable and should be avoided.